benchmarks/allocator_stress_test.cpp

```diff
@@ -56,6 +59,18 @@ void benchmark_allocator(std::string name) {
 int main(int, char**) {
   benchmark_allocator("HOST");
 
+  auto& rm = umpire::ResourceManager::getInstance();
```

Comment:
Reminder, please also update replay to be able to replay the new makeAllocator<umpire::strategy::MixedPool> operation.

Reply:
I will make this work after this has been merged into develop.